app/src/main/java/me/iacn/biliroaming/XposedInit.java

```diff
@@ -48,6 +48,7 @@ protected void beforeHookedMethod(MethodHookParam param) throws Throwable {
                         BiliBiliPackage.getInstance().init(lpparam.classLoader, (Context) param.args[0]);
                         new BangumiSeasonHook(lpparam.classLoader).startHook();
                         new CustomThemeHook(lpparam.classLoader).startHook();
+                        new BangumiPlayUrlHook(lpparam.classLoader).startHook(); // After 5.50
```

Comment:
For new version clients, the `com/bilibili/lib/okhttp/huc/OkHttpURLConnection` locates on the main process.